cpp/tests/point_in_polygon/point_in_polygon_test.cu

```diff
@@ -90,6 +122,24 @@ TYPED_TEST(PointInPolygonTest, OnePolygonOneRing)
                      {false, false, false, false, true, true, true, true});
 }
 
+TYPED_TEST(PointInPolygonTest, OnePolygonOneRingSpherical)
+{
+  CUSPATIAL_RUN_TEST(this->run_spherical_test,
+                     {{-2503.357, -4660.203, 3551.245},
```

Comment:
What CRS is this coordinate?

Reply:
Those are projections that consider the world as a sphere.

Reply:
Ah I see, I remember you said this is the (X, Y, Z) coordinate using the center of earth as (0, 0, 0). For this algorithm it shouldn't matter how you choose the orthonormal basis as long as they are consistent. It would be nice to mention this in the docstring.